### DISCUSSION TITLE
replace database driver (from driver manager) for existing connection

### DISCUSSION THREAD

**Discussion by ltdeta** [dbeaver/dbeaver#34391] [Q&A] [answered]
I created a new driver in the Driver Manager ( SQL Server V11.2.3 jre17 ) . 
Now I want to assign the new driver to an existing database connection 
(connection name "KOFAXRPA (SQLSERVER) DRV-V11.2.3 jre17" current driver "SQL Server V12.6.2 jre11"). 
How do I do that?
I can edit the driver (register library), but I would like to **replace** the driver with the newly created one.

![driver manager](https://github.com/dbeaver/dbeaver/assets/1563652/7f0201ad-3cc4-414d-8038-cf1ec34554e0)

![connection - driver - change](https://github.com/dbeaver/dbeaver/assets/1563652/a2db2cd9-d54a-475a-a7e4-923f3e57157e)

**Comment by E1izabeth** [comment]
1. Click on the connection in navigator
2. Open Database menu and choose Change connection(s) driver
![image](https://github.com/dbeaver/dbeaver/assets/28875055/7702b70a-a198-4d88-aeb8-a4f8b65bce95)

**Comment by ltdeta** [comment]
"Ok, now I've found the 'problem.' 
By default, the gallery view is selected, which only displays the database types. 

Only when you are using the "all" button or switch to the classic (list view) all available drivers displayed.

![3](https://github.com/dbeaver/dbeaver/assets/1563652/c8103a85-c96f-421b-96ef-c1c58633ca88)

![2](https://github.com/dbeaver/dbeaver/assets/1563652/2277870a-0830-4524-b611-16ebb41b04e9)

**Reply by E1izabeth** [reply, marked as answer]
This view shows you the last opened tab, when you used this dialog (for creating connection, for example)
